### ISSUE TITLE
text overlap

### ISSUE THREAD

**Issue by star-buck** [blue-systems/plasma-5.9#233]
![Screenshot_20200723_171053](https://user-images.githubusercontent.com/840070/88303588-7e8f3100-cd07-11ea-8d53-1abfd971e830.png)

**Comment by star-buck** [comment]
it doesnt even work, one cannot drag the resize handle... who thought this is a good ui design is beyond me though, its one of the worst redesifn decisions ever (and not even consistent)

**Comment by Pointedstick** [comment]
Yeah, we could maybe move this content out of a pop-up and into a sub-page. We might get complaints that it's mostly empty  though. Perhaps it could be a Kirigami overlaysheet instead. That might be a better compromise.

**Comment by Pointedstick** [comment]
Working on this.

**Comment by Pointedstick** [comment]
https://invent.kde.org/plasma/plasma-workspace/-/merge_requests/170
